client/js/jsHead.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env browser */
+import FontFaceObserver from 'fontfaceobserver';
+
+const fontASubset = new FontFaceObserver('Wellcome Bold Web Subset', {weight: 'bold'});
+const fontA = new FontFaceObserver('Wellcome Bold Web', {weight: 'bold'});
+const fontB = new FontFaceObserver('Helvetica Neue Light Web');
+const fontC = new FontFaceObserver('Helvetica Neue Medium Web');
+const fontD = new FontFaceObserver('Lettera Regular Web');
+const rootElement = document.documentElement;
+if (sessionStorage.fontsLoaded) {
```

Comment:
At the moment error. We need to introduce a cutting the mustard test for all the javascript